gfx/ChestHole16Tileset/tile_config.json

```diff
@@ -18869,7 +18869,7 @@
           "rotates": false
         },
         {
-          "id": "45_casing",
+          "id": ["45_casing", "357sig_casing"],
           "fg": 58,
```

Comment:
Why not base it off bottleneck 7.62x25?

Reply:
Oh, so the 7.62x25 is using the 120mm tank shell canister sprite.

Reply:
Here's a lineup: 9mm 7 62x25mm 357 sig 10mm 45sw 45gap 50ae
![9mm_7 62mm_357sig_10mm_45sw_45gap_50ae_002](https://user-images.githubusercontent.com/18560216/39960543-52256b98-55ea-11e8-9413-a22765260875.jpg)